recipes/guiqwt/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "guiqwt" %}
+{% set version = "3.0.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PierreRaybaut/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: fcad0c1745c2685d6dcc3367ba6c5fa5dbbcd5991bf005e787c36e8e49b520a2
+  fn: {{ name }}-{{ version }}.tar.gz
+
+  patches:
+    - curve.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - setuptools
+    - python=3.6.*
+    - guidata
+    - numpy
+    - cython
+  run:
+    - python=3.6.*
```

Comment:
please remove the pinning for ``python 3.6``

Reply:
If I remove it will it only build for python 3.7?

Reply:
Nope, for all python version in our ci

Reply:
Nice!